src/qudiet/circuit_library/standard_gates/toffoli_utility.py

```diff
@@ -81,4 +81,5 @@ def __init__(self, qreg: int, dims: Union[int, tuple], plus:int = 1, backend: Ba
         _unitary, lb, ub = toffoli_dm(qreg, plus, dims)
         self._acting_on = list(range(lb, ub+1)) 
         super().__init__(qreg, [dims[a] for a in self._acting_on], backend)
-        self._unitary = _unitary
\ No newline at end of file
+        self._unitary = _unitary
+        self._plus = plus
```

Comment:
had to add this property as I did not find another way to get the value of 'plus' from a Toffoli